src/evmTest/scala/io/iohk/ethereum/vm/ThrowSpec.scala

```diff
@@ -13,7 +13,7 @@ class ThrowSpec extends FreeSpec with Matchers {
 
       val result = contract.justThrow().call()
 
-      result.error shouldBe Some(InvalidOpCode) // TODO: change to invalid jump destination
+      result.error shouldBe Some(InvalidOpCode(0xfd.toByte))
```

Comment:
I'll update it to 0.4.10 (as it seems to be the recent version) and remove `^` so that we don't get unexpected test failures